projects/plugins/crm/tests/action-skip-test-php.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# Skip tests on PHP 8.5+ due to dependency constraints
```

Comment:
Should we mention which ones in the comment, to make it easier in the future to know when we can remove this?